lib/Dialect/FIRRTL/Import/FIRParser.cpp

```diff
@@ -2122,6 +2123,14 @@ ParseResult FIRStmtParser::parseExpImpl(Value &result, const Twine &message,
     if (parseRefRead(result))
       return failure();
     break;
+  case FIRToken::lp_tagExtract:
+    if (requireFeature(missingSpecFIRVersion, "tagExtract"))
+      return failure();
+    if (isLeadingStmt)
+      return emitError("unexpected tagExtract() as start of statement");
```

Comment:
I don't think I need this but its a little unclear.  Without it you get a flow checking error.